Core/Node-API/CMakeLists.txt

```
@@ -74,6 +70,9 @@ if(NAPI_BUILD_ABI)
             find_library(JAVASCRIPTCORE_LIBRARY JavaScriptCore)
             set(LINK_LIBRARIES ${LINK_LIBRARIES}
                 PUBLIC ${JAVASCRIPTCORE_LIBRARY})
+        elseif(UNIX)
+            set(LINK_LIBRARIES ${LINK_LIBRARIES}
+                PUBLIC ${JSCORE_LIBRARY})
```

nit: to be consistent with APPLE
```suggestion
                PUBLIC ${JAVASCRIPTCORE_LIBRARY})
```

---

Fixed.